components/inputs/input-fieldset.js

```diff
@@ -0,0 +1,41 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { labelStyles } from './input-label-styles.js';
+
+class InputFieldset extends LitElement {
```

Comment:
Does this need the RTLMixin, since it's pulling in the label styles that includes some RTL specific stuff?

Reply:
Good catch -- yes! Fixed.